deep_qa/data/instances/instance.py

```diff
@@ -334,14 +350,20 @@ def pad_sequence_to_length(sequence: List,
         much of the question set up. If you want to truncate from the other
         direction, you can.
         """
-
-        padded_sequence = []
-        for _ in range(desired_length):
-            padded_sequence.append(default_value())
-        sequence_length = min(len(sequence), desired_length)
-        if sequence_length != 0:
+        if truncate_from_right:
+            truncated = sequence[-desired_length:]
+        else:
+            truncated = sequence[:desired_length]
+        if len(truncated) < desired_length:
+            # If the length of the truncated sequence is less than the desired
+            # length, we need to pad.
+            padding_sequence = [default_value()] * (desired_length - len(truncated))
```

Comment:
Oh, wait, no, it's here: https://github.com/allenai/deep_qa/blob/a71589144f11fbecd82040f29624082569e68f12/deep_qa/data/instances/instance.py#L289.  Background instances don't use it.  We certainly have tests that check for padding with multiple words (https://github.com/allenai/deep_qa/blob/a71589144f11fbecd82040f29624082569e68f12/tests/data/instances/text_instance_test.py#L60-L64).  I guess this is passing because it's just the padding instances that get duplicated...  Still, could you change that test to use 5 for `num_sentence_words`, instead of 3?  That will show whether this is actually an issue.